app/assets/stylesheets/pq.scss

```diff
@@ -216,6 +217,11 @@ button, input[type="submit"] {
 .select2-arrow {display:none!important;}
 li.select2-search-field {border-bottom: none;}
 
+.select2-container--default .select2-selection--single,
+.select2-container--default .select2-selection--multiple { border-radius: 0px; }
+.select2-container--default.select2-container--focus .select2-selection--multiple { border: 1px solid #aaa; }
```

Comment:
Just a suggestion for lines 220 - 222, maybe utilise SASS's nesting feature to make it easier to read? (http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_rules)
```
.select2-container--default {
   .select2-selection--single, .select2-selection-multiple {
      border-radius: 0px;
   }
   &.select2-selection--focus .select2-selection--multiple { 
      border: 1px solid #aaa;
   }
}
```